geode-docs/developing/geode_apis_compatible_with_redis.html.md.erb

```diff
@@ -218,7 +184,7 @@ These commands are supported for Redis 5.
 Keys are expired in two ways, actively and passively:
 
 -   With active expiration, expiration is evaluated whenever a key is accessed. If the key is due to expire, it is deleted. Active expiration is accurate to the millisecond.
--   With passive expiration, keys are evaluated every second. If they are due to expire, they are deleted.  Passive expiration is accurate to the second.
+-   With passive expiration, keys are evaluated every 3 minutes. If they are due to expire, they are deleted.  Passive expiration is accurate to the second.
```

Comment:
I don't think it's true any longer that passive expiration is `accurate to the second` since it's only evaluated every 3 minutes... unless I'm misinterpreting what we originally meant by that?

Reply:
I had a hard time interpreting that too. My thought on it was that things will get expired every 3 minutes, as long as their expiration has been reached, at a resolution of 1 second. If you read the line before that, it says that the active expiration is accurate to the ms, but there's no guarantee it will be called every 1 ms or anything.
Do you think there's a better way to say this? It shouldn't be this difficult to parse

Reply:
I'm still not sure what it was supposed to mean originally.  What does `at a resolution of 1 second` mean?  Looks like we compare milliseconds when looking at the expiration of the timestamp and then utilize active expiration via pttl (which also uses milliseconds)to expire the key:
```
 if (entry.getValue().hasExpired(now)) {
            // pttl will do its own check using active expiration and expire the key if needed
            if (-2 == redisKeyCommands.internalPttl(entry.getKey())) {
              expireCount++;
            }
          }
```

Perhaps we could just remove it altogether for active and passive expiration if we're not sure why it was added or what it means?

Reply:
I agree with @nonbinaryprogrammer that "accurate to the second" most likely refers to the resolution of the comparison between the present time and the specified expiration time. Does the code support this interpretation?
I infer from @sabbey37 's comment that the docs could drop the "accurate to ..." clauses.

Reply:
sounds like everyone is ok with removing it, so I'm going to do that.

Reply:
Sounds good, it doesn't look like it got removed in the last commit though.

Reply:
oops forgot to push. it should be changed now